apps/api/src/database/schema/core/defs.ts

```diff
@@ -57,6 +58,8 @@ export const competitions = pgTable(
     id: uuid().primaryKey().notNull(),
     name: varchar({ length: 100 }).notNull(),
     description: text(),
+    externalLink: text(),
```

Comment:
also, do you remember why `externalLink` is required for kristal's work? not a huge issue, but idk if it's needed long-term once the app is live and competition landing pages are hosted in-app vs external links.

Reply:
For sure - @KGmajor mentioned that we'd probably have an associated landing page elsewhere in our website assets that provides a more detailed overview of the competition's parameters, etc, and this should be associated with the competitions.

For the mini-app i'm building, for example, i'll need it so when i pull and display upcoming competitions, i have somewhere to link to.